ota-plus-web/app/reactapp-new/src/theme.ts

```diff
@@ -0,0 +1,14 @@
+import { DefaultTheme } from 'styled-components';
+
+const theme: DefaultTheme = {
+  palette: {
+    primary: '#00B6B2',
```

Comment:
Can we define colors in _variables.scss instead?

Reply:
Defined style constants in `constants/styleConstants.ts`